src/gui/Src/Gui/SettingsDialog.ui

```diff
@@ -33,7 +33,7 @@
       <bool>true</bool>
      </property>
      <property name="currentIndex">
-      <number>0</number>
+      <number>3</number>
```

Comment:
Minor nitpick here :)

Reply:
Fixed, didn't even know this was a thing :P